backend/inspirehep/logger/config.py

```diff
@@ -55,3 +57,19 @@
     wrapper_class=structlog.stdlib.BoundLogger,
     cache_logger_on_first_use=True,
 )
+
+# Celery logging
+# ==============
+@task_prerun.connect()
+def log_task_context(sender, task_id, task, *args, **kwargs):
+    structlog.threadlocal.bind_threadlocal(task_id=task_id, task=task.name)
+
+
+@task_postrun.connect()
+def remove_task_context_logging(*args, **kwargs):
+    structlog.threadlocal.clear_threadlocal()
```

Comment:
@benjamin-bergia we could also export Celery metrics from workers directly by plugging into these signals